mafContrib/kneMetrics.py

```diff
@@ -17,14 +20,15 @@ class KN_lc(object):
     Parameters
     ----------
     file_list : list of str (None)
-        List of file paths to load. If None, loads up all the files from data/tde/
+        List of file paths to load. If None, loads up all the files from data/bns/
     """
-
     def __init__(self, file_list=None):
-
         if file_list is None:
             sims_maf_contrib_dir = os.getenv("SIMS_MAF_CONTRIB_DIR")
-            file_list = glob.glob(os.path.join(sims_maf_contrib_dir, 'data/bns/*.dat'))
+            # Get files, model grid developed by M. Bulla
+            # FIXME change path to data
+            ##file_list = glob.glob(os.path.join(sims_maf_contrib_dir, 'data/bns/*.dat'))
+            file_list = glob.glob('data/bns/*.dat')
```

Comment:
I believe this should also be testable on Datalab now :)

Reply:
Yes, fixed and all good.